<launch>

  <arg name="limb"     default="right"/>
  <arg name="method"   default="color"/>

  <include file="$(find baxter_demos)/config/baxter_ws_params.xml" />
  <!-- Start the Joint Trajectory Action Server -->
  <node name="rsdk_joint_trajectory_action_server" pkg="baxter_interface"
  type="joint_trajectory_action_server.py" args="--limb $(arg limb)"
  required="true"/>

  <node pkg="baxter_demos" name="object_finder" type="object_finder.py"
  args="--limb $(arg limb) --method $(arg method)" required="true"/>

</launch>
